ui/public/locales/en.json

```diff
@@ -340,9 +340,9 @@
 "label.associatednetworkid": "Associated network ID",
 "label.associatednetworkname": "Network name",
 "label.asyncbackup": "Async backup",
+"label.attaching": "Attaching",
 "label.authentication.method": "Authentication Method",
 "label.authentication.sshkey": "System SSH Key",
-"label.automigrate.volume": "Auto migrate volume to another storage pool if required",
```

Comment:
Is this removed intentionally? Is this string not used in UI? @vishesh92

Reply:
This was duplicated in L353 with the same key and value.